src/test/java/entities/ExporterTest.java

```diff
@@ -54,45 +63,106 @@ public ExporterTest() {
+                items);
+        Exporter exporter = new Exporter(inventoryname, wrapper);
         exporter.export();
-        Path p1 = Path.of("src/main/java/exports/user_inventory.txt");
+        Path p1 = Path.of("src/main/java/exports/" + inventoryname + ".txt");
         File f1 = p1.toFile();
-        Path p2 = Path.of("src/main/java/exports/serializable_inventory.txt");
+        Path p2 = Path.of("src/main/java/exports/serializable_" + inventoryname + ".txt");
         File f2 = p2.toFile();
 
         Assertions.assertTrue(f1.exists());
         Assertions.assertTrue(f2.exists());
 
+    }
+
+    /**
+     * Tests the exporter direcly by feeding a string and a wrapper.
+     * reads the first line of the file to check for correctness
+     * @throws IOException as required by Exporter
+     */
 
+    @Test
+    public void AssertUsrInvCorrect() throws IOException {
+        ExportDataWrapper wrapper = new ExportDataWrapper(inventoryViewModel.getItemList(),
+                items);
+        Exporter exporter = new Exporter(inventoryname, wrapper);
+        exporter.export();
+        Path p1 = Path.of("src/main/java/exports/" + inventoryname + ".txt");
+        File f1 = p1.toFile();
+        BufferedReader reader = new BufferedReader(new FileReader(f1));
+        String line = reader.readLine();
+        line = reader.readLine();
+        System.out.println(line);
+        String expected = "             bananas                 10.0                12345                  2.0                  3.0                    5                   12 ";
+
+        Assertions.assertEquals(expected, line);
     }
+
+    /**
+     * Test the inventory View Model to see if the inventory table contains the right information
+     */
     @Test
-    public void AssertPresenterWindow() throws IOException {
-        ExportController controller = new ExportController(inventory);
+    public void InventoryViewModel() {
+        String[][] inventoryTable = new String[items.size()][7];
+        for (int i = 0; i < items.size(); i++) {
+            InventoryItem item = (InventoryItem) items.get(i);
+            inventoryTable[i] = new String[]{item.getName(), item.getQuantity() + "", item.getBarcode(),
+                    item.getBuyPrice() + "", item.getSellPrice() + "",
+                    Integer.toString(item.getCaseQuantity()), item.getDepartment()};
+        }
+        Assertions.assertEquals(inventoryTable[0][0], "bananas");
+    }
+
+
+    /**
+     * Test to check if ExportDataWrapper can be instantinted direclty
+     */
+    @Test
+    public void ExportDatWrapper() {
+        ExportDataWrapper wrapper = new ExportDataWrapper(inventoryViewModel.getItemList(),
+                items);
+        inventoryTable = wrapper.getInventorytable();
+        wrapper.setInventorytable(inventoryTable);
+        ArrayList<Object> array = wrapper.getItems_list();
+
+        Assertions.assertEquals(11, array.size());
 
     }
```

Comment:
You should also create a test for your export use case interactor to test if it correctly turns the inventory data into data that is ready to export.